app-crypt/hashcat/hashcat-9999.ebuild

```diff
@@ -34,7 +149,24 @@ DEPEND="app-arch/lzma
 	!video_cards_nvidia? ( virtual/opencl )"
 RDEPEND="${DEPEND}"
 
+src_unpack() {
+	if [[ ${PV} == "9999" ]]; then
+		git-r3_src_unpack
+	else
+		default
+	fi
+	cargo_src_unpack
+}
+
 src_prepare() {
+	if [[ ! ${PV} == "9999" ]]; then
+		# MAINTAINER NOTE: Hashcat's build system (src/bridges/*.mk) uses '|| true'
+		# for Cargo calls. We strip these to make failures visible to emake.
+		# This is not present in master
```

Comment:
If this is fixed in git, then you don't need an `if [[ ${PV} == 9999 ]]`-style conditional here, as the live ebuild will be copied to form the next release, but it's okay to make "future" changes in here.